HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -192,5 +329,7 @@ def customizeHLTforCMSSW(process, menuType="GRun"):
     
     process = customizeHLTfor47611(process)
 
+    process = customizeHLTfor48921(process)
```

Comment:
Why is there a change in HLTrigger/Configuration/python/customizeHLTforCMSSW.py gven this PR claims to be for Phase2? Changes to Phase2 are put into the modules directly!

Reply:
These changes are specific for the Phase1 configuration, since there is only one CA, shared between Phase1 and Phase2. We have modified a few CA parameters to unify a few of cuts and added a couple more. This customisation is required to propagate these changes to the Phase1 PP and HI CA modules

Reply:
The physics outcome is exactly the same since we made sure to set all the changed parameters to the defaults used in Run3, and we verified that no changes were observed with a Run3 workflow on a TTbar RelVal with 60PU ([highStats plots](https://lferragi.web.cern.ch/plots/tracking_phase2/CAExtPR/run3/)). We do not expect to backport this PR